### PR TITLE
WIP No internet/server connection warning 

### PR DESCRIPTION
**Pull request by mateuszpmroz**

### CODE REVIEW

File: src/state/status/sagas.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+    return () => {
+      eventListener();
+    };
+  });
+}
+
+export function* updateServerConnectionStatusSaga(action: UpdateServerConnectionStatusAction | unknown) {
+  while (true) {
+    yield delay(10000);
```
Comment (mateuszpmroz) [inline src/state/status/sagas.ts:31]: not sure if that's the best way to check server status

@KedziaPawel do you have any ideas?

Comment (KedziaPawel) [reply src/state/status/sagas.ts:31]: you should subscribe to events in electrum-client like onClose, onConnect, reconnect etc.

**Comment by KedziaPawel** [comment]
Remember to handle reconnection, loadWallets because there could be some transactions made during no connection, fetch block height and resubscribe to scriptHashes because the listeners are removed onClose
Zrzut ekranu 2020-10-23 o 11 50 56

**Comment by KedziaPawel** [comment]
done in the other PR

**Pull request closed without merging**